patching/plugin-commands-patching/src/patch.ts

```diff
@@ -86,6 +87,15 @@ export async function handler (opts: PatchCommandOptions, params: string[]): Pro
 
   await writePackage(patchedDep, editDir, opts)
 
+  writeEditDirState({
+    editDir,
+    modulesDir: opts.modulesDir ?? 'node_modules',
+    value: {
+      selector: params[0],
+      applyToAll: patchedDep.applyToAll,
+    },
+  })
```

Comment:
it is really hard to understand what is happening just by looking at this code. This might improve it:

```suggestion
  savePkgPatchingDirLocation({
    editDir,
    modulesDir: opts.modulesDir ?? 'node_modules',
    patchedPkg: params[0],
    applyToAll: patchedDep.applyToAll,
  })
```

Reply:
https://github.com/pnpm/pnpm/pull/8337/commits/cbc9fc5e0e4cb0065f7b7ec253d8333420c8644c